india_compliance/public/gst_india/purchase_reco_tool/number_card.js

```diff
@@ -0,0 +1,72 @@
+frappe.provide("ic");
+
+ic.NumberCardManager = class NumberCardManager {
+	constructor(opts) {
+		Object.assign(this, opts);
+		this.make_cards();
+	}
+
+	make_cards() {
+		this.$purchase_reco_tool_cards.empty();
+		this.$cards = [];
+		this.$summary = $(`<div class="report-summary"></div>`)
+			.hide()
+			.appendTo(this.$purchase_reco_tool_cards);
+		var card_data = [
+			{
+				value: this.tax_diff,
+				label: "Tax Difference",
+				datatype: "Currency",
+				currency: this.currency,
+			},
+			{
+				value: this.taxable_value_diff,
+				label: "Taxable Value Difference",
+				datatype: "Currency",
+				currency: this.currency,
+			},
+		];
+
```

Comment:
can be a part of options provided to this. Only then will this call be generic and reusable.